File: SQLiteCodekata/app/src/main/java/com/example/bob/sqlitecodekata/MainActivity.java
package com.example.bob.sqlitecodekata;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        getSupportFragmentManager().beginTransaction()
                .add(R.id.fragment_one_container, new FragOne(), "FragOne")
                .add(R.id.fragment_two_container, new FragTwo(), "FragTwo")
                .commit();
    }
}

File: MoreCodeKataForFragment/app/src/main/java/com/example/bob/morecodekataforfragment/MainActivity.kt
package com.example.bob.morecodekataforfragment

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        supportFragmentManager.beginTransaction()
                .add(R.id.fragment_container_one, FragOne())
                .add(R.id.fragment_container_two, FragTwo())
                .commit()
    }
}
